### PR TITLE
Update documentation for 0.9.11

### LINKED ISSUE #934: Update documentation for 0.9.11
- [x] Environment variables
- [x] Skip

### PR DESCRIPTION
**Pull request by TadCordle**
Merge after 0.9.11 is released. Fixes #934

### CODE REVIEW

File: jib-gradle-plugin/README.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ Property | Type | Default | Description
 `format` | `String` | `Docker` | Use `OCI` to build an [OCI container image](https://www.opencontainers.org/).
 `useCurrentTimestamp` | `boolean` | `false` | By default, Jib wipes all timestamps to guarantee reproducibility. If this parameter is set to `true`, Jib will set the image's creation timestamp to the time of the build, which sacrifices reproducibility for easily being able to tell when your image was created.
 `entrypoint` | `List<String>` | *None* | The command to start the container with (similar to Docker's [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint) instruction). If set, then `jvmFlags` and `mainClass` are ignored.
+`environment` | `Map<String, String>` | *None* | Key-value pairs for setting environment variables on the container (similar to Docker's [ENV](https://docs.docker.com/engine/reference/builder/#env) instruction).
```
Comment (chanseokoh) [inline jib-gradle-plugin/README.md]: It'd be nice to say a bit more about the syntax, especially when giving multiple values. Is it comma-separated like `var1=value1,var2=value2`?

**Comment by coollog** [comment]
Not specifically this PR, but do you think it'd be better if we ordered the configuration parameters in alphabetical order in the tables?

File: jib-gradle-plugin/README.md (hunk 1 above)
Comment (chanseokoh) [reply jib-gradle-plugin/README.md]: Alright, that I think is enough.

**Comment by TadCordle** [comment]
@coollog Yeah, I was thinking the same thing. Might as well do it in this PR.

**Comment by TadCordle** [comment]
@GoogleContainerTools/java-tools We should get this reviewed since we just released 0.9.11

**Comment by coollog** [comment]
We should include the `to.tags` and the `container.appRoot` here as well.

**Comment by coollog** [comment]
And also might as well update the versions and CHANGELOG in this PR too.

**Pull request merged**